jpf-symbc/src/examples/TestChocoIntRangeAndTimeout.java

```diff
@@ -0,0 +1,18 @@
+import org.sosy_lab.sv_benchmarks.Verifier;
+
+public class TestChocoIntRangeAndTimeout {
+    public static void main(String[] args) {
+        int x = Verifier.nondetInt();
+        test(x);
+    }
+
+    public static void test(int x) {
+        int z = x + x;
+        if(x < z) {
+            assert false;
+            System.out.println("Branch F001");
+        } else {
+            System.out.println("Branch F002");
+        }
+    }
+}
```

Comment:
Can we have these tests and their corresponding configuration files under
examples/solvers/choco. You'll probablly add more solvers and more tests as we progress.